docs/reference/connector/apis/create-connector-api.asciidoc

```diff
@@ -43,6 +43,8 @@ DELETE _connector/my-connector
 
 * To sync data using connectors, it's essential to have the Elastic connectors service running.
 * The `service_type` parameter should reference an existing connector service type.
+//// the restrictions of service_type should be better explained
```

Comment:
The valid service_type values should be documented (or at least way to get the values) to accommodate the Kibana-less experience.